src/test/regress/sql/multi_mx_create_type.sql

```diff
@@ -0,0 +1,17 @@
+-- Tests related to create type
+
+CREATE TYPE create_order_delegator_t AS (
+     out_w_tax decimal(4, 4),
+     out_d_tax decimal(4, 4),
+     out_o_id integer,
+     out_c_discount decimal(4, 4),
+     out_c_last varchar(16),
+     out_c_credit char(2),
+     out_o_entry_d timestamp
+);
+
+\d create_order_delegator_t
+
+\c - - - :worker_1_port
```

Comment:
@halilozanakgul  how compatible is this with your changes to run our tests against hyperscale? Guess we can't connect to workers anymore.

Reply:
It should still be compatible, we have plenty of tests which connect to workers

Reply:
We have ways of connecting to workers when running Hyperscale tests, though we won't include MX tests there for now.